UnityProject/Assets/Scripts/Objects/Wallmounts/PublicDepartmentTerminal.cs

```diff
@@ -0,0 +1,161 @@
+﻿using Communications;
+using Systems.Electricity;
+using UnityEngine;
+using UI.Objects.Wallmounts;
+using System.Collections.Generic;
+using Mirror;
+
+namespace Objects.Wallmounts
+{
+	public struct MessageData
```

Comment:
I think it would be good to nest this under `PublicDepartmentTerminal`. Having a "MessageData" for all wall mounts is a bit confusing.

Reply:
GUI_TerminalMessageEntry.cs needs it I believe, can't remember.